plugins/modules/aci_rest.py

```diff
@@ -447,7 +463,16 @@ def main():
         output_path = aci.params.get("output_path")
         if output_path is not None:
             with open(output_path, "a") as output_file:
-                json.dump([payload_output_file], output_file)
+                if rest_type == "json":
+                    json.dump([payload_output_file], output_file)
+                    output_file.write("\n")
+                elif rest_type == "xml":
+                    if aci.params.get("output_format") == "json" and HAS_XML_TO_DICT:
+                        json.dump([xmltodict.parse(payload)], output_file)
```

Comment:
Can you compare the JSON output from this conversion with the JSON export from APIC for the same XML object? I am afraid it is not the same as APIC uses attributes and children keys which this will probably not use.